### PR TITLE
Event and alarm management: Add tables for event and alarms.

### PR DESCRIPTION
**Pull request by bhaveshdell**
Why I did it
This PR contains code changes for providing extension to the Event Framework as specified in the https://github.com/sonic-net/SONiC/pull/1409

How I did it
Followed design specified in https://github.com/sonic-net/SONiC/pull/1409.
Add new tables for events and alarms.

### CODE REVIEW

**Comment by shyam77git** [comment]
@bhaveshdell 
This seems to be only 1 file change I see in this code PR.
Where are all other changes? PRs? could you please list them in HLD PR: [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409).

**Comment by shyam77git** [comment]
> > @bhaveshdell This seems to be only 1 file change I see in this code PR. Where are all other changes? PRs? could you please list them in HLD PR: [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409).
> 
> The PRs are listed in [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409)
> 
> #852 [sonic-net/sonic-buildimage#17949](https://github.com/sonic-net/sonic-buildimage/pull/17949)

OK, so in all these two PRs to be reviewed? any more upcoming PR?
could you add me as reviewer to them both? Add others too so that review can be expedited

**Comment by bhaveshdell** [comment]
> > > @bhaveshdell This seems to be only 1 file change I see in this code PR. Where are all other changes? PRs? could you please list them in HLD PR: [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409).
> > 
> > 
> > The PRs are listed in [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409)
> > #852 [sonic-net/sonic-buildimage#17949](https://github.com/sonic-net/sonic-buildimage/pull/17949)
> 
> OK, so in all these two PRs to be reviewed? any more upcoming PR? could you add me as reviewer to them both? Add others too so that review can be expedited

@shyam77git Only these 2 PRs.  Repo owner can add reviewer.

**Comment by shyam77git** [comment]
> > > > @bhaveshdell This seems to be only 1 file change I see in this code PR. Where are all other changes? PRs? could you please list them in HLD PR: [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409).
> > > 
> > > 
> > > The PRs are listed in [sonic-net/SONiC#1409](https://github.com/sonic-net/SONiC/pull/1409)
> > > #852 [sonic-net/sonic-buildimage#17949](https://github.com/sonic-net/sonic-buildimage/pull/17949)
> > 
> > 
> > OK, so in all these two PRs to be reviewed? any more upcoming PR? could you add me as reviewer to them both? Add others too so that review can be expedited
> 
> @shyam77git Only these 2 PRs. Repo owner can add reviewer.

@prgeor / @zhangyanzhao - please add reviewers to this HLD.
This is a per-requisite for FM infra HLD: https://github.com/sonic-net/SONiC/pull/1527

**Comment by jeff-yin** [comment]
@qiluo-msft  @liuh-80 @shyam77git Please assign reviewers. Also, the build checks seem to be failing for reasons not related to the changes in this PR. Can you please check?

**Review by venkatmahalingam — APPROVED** [review]

**Comment by bhavini-gada** [comment]
@qiluo-msft @zbud-msft - Can we please get approval/merge on this PR?

File: common/schema.h [hunk 1]
```diff
@@ -23,6 +23,7 @@ namespace swss {
 #define CHASSIS_APP_DB      12
 #define CHASSIS_STATE_DB    13
 #define APPL_STATE_DB       14
+#define EVENT_DB            19
```
Comment (qiluo-msft) [inline common/schema.h:26]: There is some redis configuration
```
# Set the number of databases. The default database is DB 0, you can select
# a different one on a per-connection basis using SELECT <dbid> where
# dbid is a number between 0 and 'databases'-1
databases 16
```

If the max number is less than 19, will your feature work? Could you break the build prcoess if it happen?

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**